### ISSUE TITLE
Accessibility requirements for Sidebar

### ISSUE THREAD

**Issue by dhruv-d-simform** [dhruvdhaduk-simform/x-skeleton-ui#11]
- Reference : https://github.com/dhruvdhaduk-simform/x-skeleton-ui/pull/10
- Sections in the sidebar needs headings, but `h1` is never used anywhere in the page. Therefore, we cannot just start directly from `h2`.
- Using `div` with `role="heading"` and `aria-level` is warned by markup validator :-
> [!WARNING]
> Section lacks heading. Consider using h2-h6 elements to [add identifying headings to all sections](https://www.w3.org/wiki/HTML/Usage/Headings/Missing), or else use a div element instead for any cases where no heading is needed.

- Though the PR is merged for now., the issue persists. And it will be fixed later. 
- For future reference, I am not deleting the branch.
